assets/src/features/home-page/components/VideoroomHomePage.tsx

```diff
@@ -59,12 +59,20 @@ const VideoroomHomePage: FC = () => {
 
   const navigate = useNavigate();
   const { audio, video } = useLocalPeer();
+  const wasVideoStarted = useRef(false);
+  const wasAudioStarted = useRef(false);
 
   useEffect(() => {
-    if (!video.stream) video.start();
-    if (!audio.stream) audio.start();
-  }, []);
-  
+    if (!wasVideoStarted.current && !video.stream) {
+      video.start();
+      wasVideoStarted.current = true;
+    }
+    if (!wasAudioStarted.current && !audio.stream) {
+      audio.start();
+      wasAudioStarted.current = true;
+    }
+  }, [audio.stream, video.stream]);
```

Comment:
We need to listen to the streams, because when the user returns to the home page straight from the room page, some time elapses before the cleanup function inside `ConnectComponent` stops the camera, and we might end with a stale stream here.